demo/runners/performance.py

```diff
@@ -760,6 +764,13 @@ async def check_received_pings(agent, issue_count, pb):
             "('debug', 'info', 'warning', 'error', 'critical')"
         ),
     )
+    parser.add_argument(
+        "--cred-type",
+        type=str,
+        metavar="<cred-type>",
+        default=None,
+        help=("Specifyng the credential type"),
```

Comment:
good idea to have this simple flag.  is there a default credential type that is used if the flag is not present?  should we use that as the default for the flag value to give the user a clue about that behavior

Reply:
The credential type is defaulted when creating the agent:

`cred_type: str = CRED_FORMAT_INDY,`

... it would probably be more clearer to assign the default in the input argument tho ...